sem/utils.py

```diff
@@ -565,20 +564,29 @@ def parse_logs(log_file):
             # group[4] = PowerAdaptationDistance
             # group[5] = SetupPhy
             # group[6] = ''
-            # group[7] = : [DEBUG] OfdmRate6Mbps 0.00192 6000000bps
-            # group[8] = DEBUG
-            # group[9] = OfdmRate6Mbps 0.00192 6000000bps
+            # group[7] = DEBUG
+            # group[8] = OfdmRate6Mbps 0.00192 6000000bps
             groups = regex.match(log)
 
             if groups is None:
                 warnings.warn("Log format is not consistent with prefix_all. Skipping log '%s'" % log, RuntimeWarning)
                 continue
 
+            # If severity class is 'function', then arguments cannot be empty
+            if groups[7] is None and groups[8] is None and groups[6] is None:
+                warnings.warn("Log format is not consistent with prefix_all. Skipping log '%s'" % log, RuntimeWarning)
+                continue
```

Comment:
could you explain? if the severity is function those three groups should indeed be empty, shouldn't they?

Reply:
If the severity level of a log is function, then the 'arguments' cannot be empty. This check ensures that condition.